src/test/web-platform-tests/manifests/idbobjectstore-deleteIndex-exception-order.any.toml

```diff
@@ -2,8 +2,8 @@
 ["IDBObjectStore.deleteIndex exception order: InvalidStateError #2 vs. TransactionInactiveError"]
 expectation = "UNSTABLE"
 
-["IDBObjectStore.deleteIndex exception order: InvalidStateError #1 vs. TransactionInactiveError"]
+["IDBObjectStore.deleteIndex exception order: TransactionInactiveError vs. NotFoundError"]
 expectation = "UNSTABLE"
 
-["IDBObjectStore.deleteIndex exception order: TransactionInactiveError vs. NotFoundError"]
+["IDBObjectStore.deleteIndex exception order: InvalidStateError #1 vs. TransactionInactiveError"]
 expectation = "UNSTABLE"
```

Comment:
The order changed here due to `GENERATE_MANIFESTS=1` – I opted to keep it so that future uses of this flag wouldn't mutate the file.